docs/html/installation.md

```diff
@@ -67,9 +67,17 @@ $ python pip.pyz --help
 
 If run directly:
 
-```{pip-cli}
-$ pip.pyz --help
+````{tab} Unix/macOS
+```console
+$ ./pip.pyz
+```
+````
+
+````{tab} Windows
+```console
+C:\> .\pip.pyz
 ```
+````
```

Comment:
On windows this needs to register the .pyz extension and associate it to some python interpreter? Does this support virtual environments?

Reply:
> On windows this needs to register the .pyz extension and associate it to some python interpreter?

Yeah, that's done during installation. Considering that it's based on `py` launcher, I'm guessing it's only done when that is installed.

> Does this support virtual environments?

Yes, `py` launcher uses the activated virtual environment *if* a specific Python version isn't specified, which it isn't when running the `.pyz` file directly.

---

During my testing, I found out that zipapps don't work as great on Windows as they do on Unix since they're only based on the registered extensions and therefore aren't recognized transparently everywhere (e.g. `subprocess.run(["pip.pyz"])` doesn't work, running it in PowerShell causes Command Prompt window to open instead of using the existing terminal).

Reply:
> zipapps don't work as great on Windows as they do on Unix

Correct. On Windows, I would typically recommend using `py C:\Path\To\pip.pyz`, or using a launcher executable. Users of the `scoop` package manager on Windows can use something like https://github.com/pfmoore/scoop-enk/blob/master/bucket/pip.json (or just register that bucket and use it directly 🙂)

Reply:
Can this be fixed to say `py .\pip.pyz`?

Reply:
Just to make sure I understand you correctly, you want both of these invocations to use `py pip.pyz`?
![image](https://github.com/pypa/pip/assets/6032823/1d2026a7-11ff-47f6-b649-d707b5a22966)

Instead of being different how it's on Linux/MacOS:
![image](https://github.com/pypa/pip/assets/6032823/299d9096-0a57-43e3-ab17-e7095622f10a)

Such change would be fine for me, I just want to make sure that's what you want. In such a case, I should probably add some text before the command on the second Windows tab to minimize the user confusion though. Or maybe keep only a single Windows tab for that whole text there and only show a single command on the Windows tab?

Reply:
Ah, that's a good point. I hadn't spotted that this was the "if run directly" section. I honestly don't know how we explain what's needed simply here. You're correct that it *shouldn't* use `py` as that's not "running it directly".

I don't think the pip documentation is the right place to get into how to set your system up to automatically run `.pyz` files, but I *also* don't think we're doing anyone any favours by giving a command that won't work for a non-trivial subset of users 🙁

Maybe we just remove the example invocation altogether, just leaving the statement "If run directly then the currently active Python interpreter will be used". It's a little cryptic, leaving the user to infer what "running directly" means, but at least it isn't flat-out incorrect.

Reply:
I committed what I think would be the least cryptic option. It isn't quite what you proposed so let me know what you think about that:
https://pip--12043.org.readthedocs.build/en/12043/installation/#standalone-zip-application

Reply:
Personally, I don't like it, as the way the second example disappears when I switch from Linux to Windows is extremely disconcerting.

OK, this all sucks. I'm going to say let's just go back to using plain `.\pip.pyz` on Windows, and hope we don't get too many confused Windows users asking us how to register the `.pyz` extension. I don't like it, but all of the alternatives seem worse to me.

Maybe we just add a note - something like

> If run directly:
> `C:> .\pip.pyz --help`
> then the currently active Python interpreter will be used. You may need to configure your system to recognise the `.pyz` extension before this will work.

that would be enough?

By the way, I just noticed that the Windows version, `C:> py pip.pyz --help` renders in italics, for some weird reason. It shouldn't do that. I have no idea why it happens - other console commands don't display like that.

Reply:
> Maybe we just add a note - something like
> 
> > If run directly:
> > `C:> .\pip.pyz --help`
> > then the currently active Python interpreter will be used. You may need to configure your system to recognise the `.pyz` extension before this will work.
> 
> that would be enough?

Seems fine to me, will change it to that in a moment.

> By the way, I just noticed that the Windows version, `C:> py pip.pyz --help` renders in italics, for some weird reason. It shouldn't do that.

That's partially on me, I tried to make it consistent with the prefix used by `{pip-cli}` and I removed the `\` from `C:\>` which seems to get rendered differently. `C:\>` is actually more correct anyway since that's how Command Prompt shows the root drive path so I'll change it back.

Reply:
Okay, I think it looks good now. Turns out that the italics was caused by the `console` lexer being used for the code block. I changed it to the proper `doscon`. Btw, the same problem can also be observed in user guide document.

As for the note, I added it as a separate line in the Windows tab. To me, it looks less disconcerting when switching tabs, see:
<details>
<summary>Linux tab</summary>

![chrome_h7g4UpQ64U](https://github.com/pypa/pip/assets/6032823/9ab3f39f-c7ec-4283-aa8b-67b7f34a96f0)

</details>
<details>
<summary>Windows tab with one paragraph</summary>

![chrome_RcIiEbjhPw](https://github.com/pypa/pip/assets/6032823/04bc5c87-25ef-44f6-85fe-07739e2c685f)

</details>
<details>
<summary>Windows tab with two paragraphs</summary>

![chrome_eWdTEZ5CzN](https://github.com/pypa/pip/assets/6032823/5566dca9-d4d7-45b3-807d-c184adb55e5f)

</details>